From baa0bf3d30940aab60db6a69eec7ef6d6b5bf041 Mon Sep 17 00:00:00 2001
From: Meir Levi <mlevi4@marvell.com>
Date: Mon, 18 Jan 2021 05:41:35 -0500
Subject: [PATCH 1151/1921] dts: marvell: ap80x: enable dma-coherent for
 musdk_cma

Enable dma-coherent for musdk_cma.

Signed-off-by: Meir Levi <mlevi4@marvell.com>
Change-Id: I5a6357ea1ae94b3b67d96d4b5a1ce7e8c4034f28
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/43843
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-ap80x.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi b/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
index 82a79c6765c5..7b0c008a1382 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap80x.dtsi
@@ -324,6 +324,7 @@
 
 			musdk_cma {
 				compatible = "marvell,musdk-cma";
+				dma-coherent;
 			};
 		};
 	};
-- 
2.31.1

